resources/views/components/grid/migrated-address.blade.php

```diff
@@ -0,0 +1,49 @@
+@props ([
+    'model',
+])
+
+<div class="flex justify-between items-center pb-4 border-b border-theme-secondary-300 dark:border-theme-secondary-800">
+    <div class="flex flex-col flex-1 space-y-2 min-w-0">
+        <div class="flex items-center">
+            <span class="text-sm font-semibold text-theme-secondary-500 dark:text-theme-secondary-700">
+                @lang ('general.transaction.migrated_address')
+            </span>
+        </div>
+
+        <span class="font-semibold text-theme-secondary-700">
+            <x-ark-external-link url="https://polygonscan.com/address/{{ $model->migratedAddress() }}">
+                <x-slot name="text">
+                    <span class="sm:hidden md:inline lg:hidden">
+                        <x-truncate-middle :length="12">
+                            {{ $model->migratedAddress() }}
+                        </x-truncate-middle>
+                    </span>
+
+                    <span class="hidden sm:inline md:hidden lg:inline">
+                        {{ $model->migratedAddress() }}
+                    </span>
+                </x-slot>
+            </x-ark-external-link>
+        </span>
+    </div>
+
+    <div
+        class="flex justify-center items-center w-12 h-12 rounded-full"
+        x-data="{
+            init() {
+                const canvas = window.blockies.create({
+                    seed: '{{ $model->migratedAddress() }}',
+                    color: '#fff',
+                    size: 11,
+                    scale: 4,
+                    spotcolor: 'random',
+                })
+
+                canvas.classList.add('rounded-full')
+
+                $root.appendChild(canvas)
```

Comment:
your colour issue seems to have to do with how it's appended. If you append it twice with `document.body.appendChild(canvas);` instead, you will see the first one being black and the second one having colours. Tested by running this twice in the console to get two images:

```
const canvas = window.blockies.create({
    seed: 'test',
    size: 11,
    scale: 5,
    color: 'random'
});
canvas.classList.add('rounded-full');
document.body.appendChild(canvas);
```